modules/040-control-plane-manager/openapi/config-values.yaml

```diff
@@ -205,6 +205,12 @@ properties:
     description: |
       `etcd` parameters.
     properties:
+      disableAutoDefragmentation:
```

Comment:
We decided that usage of `enable` is preferable than `disable`. Change this option to `enableAutoDefragmentation`.